### ISSUE TITLE
fitted values from fit1() with kinship are wrong

### ISSUE THREAD

**Issue by kbroman** [rqtl/qtl2#172]
[Gary](https://www.jax.org/research-and-faculty/faculty/gary-churchill) reported problems with the fitted values from `fit1()`. Indeed, you can tell that they're messed up when you include a kinship matrix, because with different shuffles of the genotype data, you get different results.

**Comment by kbroman** [comment]
It looks like those fitted values, in the case of a kinship matrix, are just completely wrong. They're [taken from the fit in `fit1_pg_addcovar()`](https://github.com/rqtl/qtl2/blob/master/R/fit1_pg.R#L163), which is in C++, but that's [taken from `fit_linreg()`using the rotated X and phenotypes](https://github.com/rqtl/qtl2/blob/master/src/fit1_pg.cpp#L65), and those need to be rotated back.
